tools/geo_converters/Dockerfile

```diff
@@ -34,7 +34,7 @@ RUN apk add perl musl
 RUN apk add cmake g++ make musl-dev && cd /usr/app/proc_gdal/src && \
   cmake . && make && mv proc_gdal /usr/app/proc_gdal && \
   cd .. && rm -rf src && apk del cmake g++ make musl-dev
-USER solr
+# USER solr
```

Comment:
You want to make this previously rootless container to root one. It will create files, owned by root.
Why do you want to do it?

Reply:
I tried to experiment around it and was surprised that things are changed and base container not rootless anymore, so there is no 'solr' user in it.
So you may go on and completely remove 'USER solr' from it. Or I'll do it later. 'USER root' better be left in place, as these guys may change their mind one more time.
If you (like me) will be annoyed by the fact that owner of all generated files is 'root' - you may 'docker run' this container on behalf of your user's UID and GIDs.

OTOH, the proper remedy for JSONArgsRecommended (that I never seen before) is 'CMD sh' -> 'CMD [sh]'